frontend/src/components/editor/chrome/wrapper/sidebar.tsx

```diff
@@ -0,0 +1,103 @@
+/* Copyright 2024 Marimo. All rights reserved. */
+import React, { PropsWithChildren } from "react";
+import { MessageCircleQuestionIcon } from "lucide-react";
+import { cn } from "@/utils/cn";
+import { useChromeActions, useChromeState } from "../state";
+import { Tooltip } from "@/components/ui/tooltip";
+import { FeedbackButton } from "../components/feedback-button";
+import { PANEL_ICONS, PanelType } from "../types";
+
+export const Sidebar: React.FC = () => {
+  const { selectedPanel } = useChromeState();
+  const { openApplication } = useChromeActions();
+
+  const renderIcon = (type: PanelType, className?: string) => {
+    const Icon = PANEL_ICONS[type];
+    return <Icon className={cn("h-5 w-5", className)} />;
+  };
+
+  return (
+    <div className="h-full py-4 px-1 flex flex-col items-start text-muted-foreground text-md border-t border-border select-none no-print text-sm z-50 absolute">
```

Comment:
Do you like the border top?

Reply:
I can't actually tell the difference with and without the border. It was a holdover from the footer — will remove.